unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -143,28 +143,33 @@ public void ConfigureHUDElement(HUDElementID hudElementId, HUDConfiguration conf
                 break;
             case HUDElementID.WORLD_CHAT_WINDOW:
                 CreateHudElement<WorldChatWindowHUDController>(configuration, hudElementId);
-                worldChatWindowHud?.Initialize(ChatController.i, DCL.InitialSceneReferences.i?.mouseCatcher);
-                taskbarHud?.AddWorldChatWindow(worldChatWindowHud);
+                if (worldChatWindowHud != null)
+                {
+                    worldChatWindowHud.Initialize(ChatController.i, DCL.InitialSceneReferences.i?.mouseCatcher);
+                    worldChatWindowHud.OnPressPrivateMessage -= OpenPrivateChatWindow;
+                    worldChatWindowHud.OnPressPrivateMessage += OpenPrivateChatWindow;
```

Comment:
The same as before. In this case we could use the OnDestroy event to unsubscribtions.

Reply:
Done